src/DGtal/geometry/curves/FrechetShortcut.h

```diff
@@ -52,6 +52,9 @@
 
 #include "DGtal/geometry/curves/SegmentComputerUtils.h"
 
+
+#define PRECISION 0.00000001
```

Comment:
Would it be possible to have this as a constructor parameter (with the same default value) ?